WalletWasabi.Gui/Tabs/WalletManager/GenerateWalletViewModel.cs

```diff
@@ -63,8 +63,6 @@ private void DoGenerateCommand()
 			WalletName = Guard.Correct(WalletName);
 
 			string walletFilePath = Path.Combine(Global.WalletsDir, $"{WalletName}.json");
-			Password = Guard.Correct(Password); // Don't let whitespaces to the beginning and to the end.
-			PasswordConfirmation = Guard.Correct(PasswordConfirmation); // Don't let whitespaces to the beginning and to the end.
```

Comment:
This should be put back. I'm lazy to argue why this is a good idea, rather I just point out that it can break existing users' passwords, which is unacceptable.